playground/public/trial-manager.mjs

```diff
@@ -0,0 +1,80 @@
+class TrialManager extends HTMLElement {
+    static observedAttributes = ['mode', 'message'];
+    constructor() {
+        super()
+    }
+
+    render() {
+        const mode = this.getAttribute('mode');
+        if (!mode) {
+            this.replaceChildren(this.initializingEl())
+        } else if (mode === 'new_session') {
+            const progress = document.createElement('div')
+            progress.classList.add('rounded-md', 'p-4', 'bg-white', 'w-96', 'text-center')
+            progress.innerText = this.getAttribute('message')
+            this.replaceChildren(progress)
+        } else if (mode === 'existing_session') {
+            this.replaceChildren(this.actionsEl())
+        }
+    }
+
+    connectedCallback() {
+        this.classList.add('flex', 'flex-col', 'items-center', 'justify-center', 'w-full', 'h-dvh')
+        this.render()
+    }
```

Comment:
Set elements as properties here. Then in render use replace with elements from properties